src/ontology/hp.Makefile

```diff
@@ -636,3 +636,30 @@ $(TMPDIR)/hp-%-merged.owl: hp-base.owl tmp/%.owl
 
 mappings: 
 	$(MAKE_FAST) ../mappings/hp-snomed.lexmatch.sssom.tsv
+
+.PHONY: diff
+diff: diff-table diff-kgcl diff-yaml
+
+tmp/hp-released.db:
+	wget http://purl.obolibrary.org/obo/hp.owl -O tmp/hp-released.owl && cp ../../hp-base.owl tmp/hp-base.owl
```

Comment:
`cp ../../hp-base.owl tmp/hp-base.owl` and `semsql make $@` does not belong here - make separate make goals. In the near future we need to be able to make diff between different versions of the ontology, such as `main` vs `branch`.

Reply:
refactored them all!